proarc-common/src/main/java/cz/cas/lib/proarc/common/export/NdkExportOptions.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2014 Jan Pokorsky, Robert Simonovsky
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package cz.cas.lib.proarc.common.export;
+
+import org.apache.commons.configuration.Configuration;
+
+/**
+ * Settings for NDK export
+ *
+ * @author lsykora
+ */
+public class NdkExportOptions {
+    static final String PROP_NDK_AGENT_CREATOR = "export.ndk.agent.creator";
+    static final String PROP_NDK_AGENT_ARCHIVIST = "export.ndk.agent.archivist";
+    private String creator;
+    static private String archivist;
+    static private NdkExportOptions options;
+
+    private NdkExportOptions() {
+    }
+
+    public static NdkExportOptions getNdkExportOptions(Configuration config) {
+        if (config == null && options != null) {
+            return options;
+        } else if (config != null && options == null) {
+            options = new NdkExportOptions();
+
+            String creator = config.getString(PROP_NDK_AGENT_CREATOR);
+            if (creator != null && !creator.isEmpty()) {
+                options.setCreator(creator);
+            }
+
+            String archivist = config.getString(PROP_NDK_AGENT_ARCHIVIST);
+            if (archivist != null && !archivist.isEmpty()) {
+                options.setArchivist(archivist);
+            }
+        }
+        return options;
+    }
+
+    /**
+     * Returns the creator organization - used for mets header
+     *
+     * @return creator
```

Comment:
Pokud parametry nekomentuješ, tak je z javadocu vynechej. Tohle jen zabírá místo a nikomu to nepomůže.